src/main/java/com/redhat/sast/api/model/JobBatch.java

```diff
@@ -53,6 +53,9 @@ public class JobBatch {
     @Column(name = "use_known_false_positive_file")
     private Boolean useKnownFalsePositiveFile;
 
+    @Column(name = "pipeline_name")
+    private String pipelineName;
```

Comment:
Why is this necessary here? JobBatch row cannot exists without a Job row. So, it feels like redundant to have it here. Isn't it? I think better approach is to keep it here and remove it from the Job table. wdyt?